openstack/02b-MAChigh/sixtop.c

```diff
@@ -175,6 +175,12 @@ void sixtop_request(uint8_t code, open_addr_t* neighbor, uint8_t numCells){
         return;
     }
    
+   if (schedule_getNumberOfFreeEntries() < numCells){

```

Comment:
What I mean is: what does the mote initiating the 6P transaction do when it receives the error code. Does it start over immediately?